dcs/task.py

```diff
@@ -739,7 +739,7 @@ def __init__(self):
 class RecoveryTanker(Task):
     Id = "RecoveryTanker"
 
-    def __init__(self, groupId: int, speed: float, altitude: float, lastWaypoint: Optional[int]):
+    def __init__(self, groupId=None, speed=600, altitude=4000, lastWaypoint: Optional[int] = None):
```

Comment:
Don't delete the type annotations. You've changed this signature from `float` to `int` here.